spacy/cli/convert.py

```diff
@@ -189,33 +189,6 @@ def autodetect_ner_format(input_data: str) -> Optional[str]:
     return None
 
 
-def walk_directory(path: Path, converter: str) -> List[Path]:
```

Comment:
I think the changes in this file should be reverted in order not to affect the `spacy convert` behavior.

Reply:
I'm not sure how is the behavior affected? The only (intended) change to `walk_directory` is that providing the `suffix` is now `Optional`.

Reply:
Ah, I thought the converter/suffix behavior was more complicated than it is (and we just don't have good tests here that would catch unintended changes).